doc/src/man/linkcheckerrc.rst

```diff
@@ -95,9 +95,13 @@ checking
     be accessed before checking.
     The default is to use robots.txt files.
     Command line option: :option:`--no-robots`
-**allowedschemes=**\ *NAME*\ [**,**\ *NAME*...]
+**allowedschemes=**\ *NAME*\ [**,**\ *NAME*...**
```

Comment:
`**` is a typo? Would be a different change anyway.

Reply:
Yes, it is. Thanks for noticing.